src/Opdex.Platform.WebApi/Auth/AdminOnlyHandler.cs

```diff
@@ -25,12 +33,15 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
     {
         if (await _featureManager.IsEnabledAsync("AuthServer"))
         {
-            if (!context.User.HasClaim(c => c.Type == "wallet")) return;
+            if (!context.User.HasClaim(c => c.Type == JwtRegisteredClaimNames.Sub)) return;
 
-            var wallet = context.User.FindFirstValue("wallet");
+            var wallet = context.User.FindFirstValue(JwtRegisteredClaimNames.Sub);
             var admin = await _mediator.Send(new GetAdminByAddressQuery(wallet, findOrThrow: false));
             if (admin is null) return;
```

Comment:
I think we can just remove this now, no need for these requirements once we're using the auth server as the admin table will be removed at that point.

Reply:
nvm based on our discussion